src/py42/clients/alerts.py

```diff
@@ -103,3 +105,56 @@ def _add_tenant_id_if_missing(self, query):
             return json.dumps(query_dict)
         else:
             return str(query)
+
+    def _get_alert_rules(
+        self, tenant_id, sort_key=None, sort_direction=None, page_num=None, page_size=None
+    ):
+        data = {
+            "tenantId": tenant_id,
+            "groups": [],
+            "groupClause": "AND",
+            "pgNum": 0,  # This API expects first page to start with zero, hence hard-coded.
+            "pgSize": page_size,
+            "srtKey": sort_key,
+            "srtDirection": sort_direction,
+        }
+        uri = self._uri_prefix.format(u"rules/query-rule-metadata")
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get_all(self, sort_key="CreatedAt", sort_direction="DESC"):
```

Comment:
Let's name this `get_all_rules()` so that we know it returns rules and not alerts, since it is now in the `AlertsClient`

Reply:
Method names changed.